hnn_core/network.py

```diff
@@ -2175,6 +2222,80 @@ def _standardize_spike_data(self, spike_data):
 
         return standardized_data, n_drive_cells, source_to_gid_map
 
+    def _get_next_available_gid(self):
+        """Return the next unused GID number, equal to current max GID + 1."""
+        max_gid = -1
+        for rng in self.gid_ranges.values():
+            if len(rng) > 0:
+                max_gid = max(max_gid, max(rng))
```

Comment:
```suggestion
        for gid_range in self.gid_ranges.values():
            if len(gid_range) > 0:
                max_gid = max(max_gid, max(gid_range))
```
Nitpick but `rng` is pretty exclusively used to mean random number generator in a lot of code